cypress/e2e/layers.cy.ts

```diff
@@ -263,165 +315,183 @@ describe("layers", () => {
-          driver.doesNotExists(errorSelector);
+          should.notExist(errorSelector);
 
-          driver.click(".CodeMirror");
-          driver.typeKeys("\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013 {");
-          driver.isExists(errorSelector);
+          when.click(".CodeMirror");
+          when.typeKeys(
+            "\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013 {"
+          );
+          should.isExists(errorSelector);
 
-          driver.click(driver.getDataAttribute("layer-editor.layer-id"));
+          when.click("layer-editor.layer-id");
         });
       });
-    })
+    });
   });
 
-  describe('fill', () => {
+  describe("fill", () => {
     it("add", () => {
-
-      var id = driver.fillLayersModal({
+      var id = when.fillLayersModal({
         type: "fill",
-        layer: "example"
+        layer: "example",
       });
 
-      driver.isStyleStoreEqual((a: any) => a.layers, [
-        {
-          "id": id,
-          "type": 'fill',
-          "source": "example"
-        }
-      ]);
-    })
+      should.equalStyleStore(
+        (a: any) => a.layers,
+        [
+          {
+            id: id,
+            type: "fill",
+            source: "example",
+          },
+        ]
+      );
+    });
 
     // TODO: Change source
-    it("change source")
+    it("change source");
   });
 
-  describe('line', () => {
+  describe("line", () => {
     it("add", () => {
-      var id = driver.fillLayersModal({
+      var id = when.fillLayersModal({
         type: "line",
-        layer: "example"
+        layer: "example",
       });
 
-      driver.isStyleStoreEqual((a: any) => a.layers, [
-        {
-          "id": id,
-          "type": "line",
-          "source": "example",
-        }
-      ]);
+      should.equalStyleStore(
+        (a: any) => a.layers,
+        [
+          {
+            id: id,
+            type: "line",
+            source: "example",
+          },
+        ]
+      );
     });
 
     it("groups", () => {
       // TODO
       // Click each of the layer groups.
-    })
+    });
   });
 
-  describe('symbol', () => {
+  describe("symbol", () => {
     it("add", () => {
-      var id = driver.fillLayersModal({
+      var id = when.fillLayersModal({
         type: "symbol",
-        layer: "example"
+        layer: "example",
       });
 
-      driver.isStyleStoreEqual((a: any) => a.layers, [
-        {
-          "id": id,
-          "type": "symbol",
-          "source": "example",
-        }
-      ]);
+      should.equalStyleStore(
+        (a: any) => a.layers,
+        [
+          {
+            id: id,
+            type: "symbol",
+            source: "example",
+          },
+        ]
+      );
     });
   });
 
-  describe('raster', () => {
+  describe("raster", () => {
     it("add", () => {
-      var id = driver.fillLayersModal({
+      var id = when.fillLayersModal({
```

Comment:
Is it possible to better type `fillLayersModal` parameter?
seems like `{ type: string, layer?: string }`...?